hi-ml-cpath/src/health_cpath/utils/viz_utils.py

```diff
@@ -108,10 +109,11 @@ def plot_attention_tiles(
     """
     tile_nodes = slide_node.top_tiles if top else slide_node.bottom_tiles
     num_rows = int(ceil(len(tile_nodes) / num_columns))
-    assert (
-        num_rows > 0
-    ), "The number of selected top and bottom tiles is too low. Try debugging with a higher num_top_tiles and/or a "
-    "higher number of batches."
+    if num_rows == 0:
+        logging.warning(
+            "The number of selected top and bottom tiles is too low. Try debugging with a higher "
```

Comment:
Can you also say that plotting will be skipped? just so that it's clear for the user